webui-src/app/mail/mail_attachment.js

```diff
@@ -1,24 +1,64 @@
 const m = require('mithril');
+const rs = require('rswebui');
 const util = require('mail/mail_util');
 
 const Layout = () => {
+  const files = [];
+  let changeView = false;
+
   return {
+    oninit: (v) => {
+      v.attrs.list.forEach(async (element) => {
+        const res = await rs.rsJsonApiRequest('/rsMsgs/getMessage', {
+          msgId: element.msgId,
+        });
+        if (res.body.retval) {
+          res.body.msg.files.forEach((element) => {
+            files.push({ ...element, from: res.body.msg.from, ts: res.body.msg.ts });
+          });
+        }
+      });
+    },
     view: (v) => [
       m('.widget', [
-        m('h3', 'Attachment'),
+        m('div.msg-attachment-container', [
+          m('h3', 'Attachment'),
+          m('.view', [
+            m(
+              '.mail-view',
+              {
+                onclick: () => (changeView = false),
+                style: { backgroundColor: changeView ? '#fff' : '#019DFF' },
+              },
+              m('i.fas.fa-mail-bulk')
```

Comment:
`view` is not very descriptive, maybe we can name it something like view-toggle or view-switch.

Also can we move the color values to scss?